Python/Tests/GlassTests/PythonTests/Python/Python.GlassTestGroup

```diff
@@ -1,40 +1,88 @@
 <?xml version="1.0" encoding="utf-8"?>
 <GlassTestGroup xmlns="http://schemas.microsoft.com/vstudio/diagnostics/glasstestmanagement/2014">
-  <Configurations>
+  <Configurations> 
     <StandardConfiguration Name="36-32" TargetArchitecture="x86">
       <Setup>
         <ImportPropertyGroup>..\Python\Python36.32.GlassTestProps</ImportPropertyGroup>
+        <ImportPropertyGroup>C:\GlassTesting\GlassStandAlone\Native\InProcPdb.GlassTestProps</ImportPropertyGroup>
```

Comment:
There shouldn't be absolute pathnames here, since other machines might not have it deployed to the same location (and it might not be possible to deploy it there, e.g. on CI). 

Where is C:\GlassTesting from, and how is somebody who loads this project on another machine expected to obtain it?

Reply:
Right. Since PythonTests (ours) and GlassStandAlone (theirs) are all under the same root, you should be able to use relative paths to go between them and never refer to c:\GlassTesting.

Reply:
Will be updated in next commit